src/PhpParser/Node/BetterNodeFinder.php

```diff
@@ -270,21 +270,6 @@ public function findFirstPrevious(Node $node, callable $filter): ?Node
         return null;
     }
 
-    /**
-     * @api
```

Comment:
As this method was api before, should we at least mark it as deprecated in a few releases before dropping it?

Reply:
`findFirstPrevious()` still available -- for now --, it basically just shortcut with only pass type instead of callable filter.

Reply:
IMO for good DX we would change api only when really required. 

Even if this method is no longer used in rector-src it might have been used by people in custom rules. Therefore I think we should keep it but mark it as deprecated for at least a few releases, before dropping it.

Reply:
I will let @TomasVotruba decide on that if it going to be reverted.

Reply:
I think this method was rarely used, so its ok. Also refactoring to parent nodes should start as soon as possible, to get other packages who use it into new shape.